test/passing/gen/gen.ml

```diff
@@ -89,8 +89,10 @@ let register_file tests fname =
 let cmd should_fail args =
   let cmd_string = String.concat " " args in
   if should_fail then
-    spf {|(with-accepted-exit-codes 1
-       (run %s))|} cmd_string
+    spf
+      {|(with-accepted-exit-codes 1
+       (run %s))|}
+      cmd_string
```

Comment:
This output can be achieved by tweaking `fmt_args_grouped` instead of adding breaks as experimented in https://github.com/ocaml-ppx/ocamlformat/pull/2448